core/domain/exp_services.py

```diff
@@ -1148,17 +1462,34 @@ def _exp_rights_to_search_dict(rights):
 
 
 def _should_index(exp):
+    """
+    Args: 
+        exp: Exploration domain object.
+    """
     rights = rights_manager.get_exploration_rights(exp.id)
     return rights.status != rights_manager.ACTIVITY_STATUS_PRIVATE
 
 
 def get_number_of_ratings(ratings):
+    """ Used to get the total number of ratings.
```

Comment:
Remove space at beginning.

Gets the total number of ratings represented by the given ratings object.